experimental/sql/dialects/iterators.py

```diff
@@ -0,0 +1,102 @@
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from dataclasses import dataclass
+from typing import Any, Union, List
+from xdsl.ir import Block, Region, Operation, SSAValue, ParametrizedAttribute, Data, MLContext, Attribute
+from xdsl.dialects.builtin import StringAttr, ArrayAttr, ArrayOfConstraint, IntegerAttr, IntegerType, TupleType
+from xdsl.irdl import AttributeDef, OperandDef, ResultDef, RegionDef, SingleBlockRegionDef, irdl_attr_definition, irdl_op_definition, ParameterDef, AnyAttr, VarOperandDef, builder
```

Comment:
No, since it would have to do non-trivial changes AFAIK. Multiple imports line-breaked would look like this e.g.:

```
from package_name import (
    x,
    y,
    z,
)
```

I can change it if you want. I feel that these lines are quite straight-forward and readable, even in this form...